### PR TITLE
Service borg shaker buff redux

### PR DESCRIPTION
**Pull request by 123chess456**
## About The Pull Request

This is the same as #3860 except there are now two service borg shakers, one for alcohol and one without. Basically this harmonizes the service borg shaker reagants with booze/soda dispensers. Service borgs can now serve the same drinks as the booze and soda dispenser in combination can. The non-alcoholic shaker is the base shaker in preparation for hypothetical Muslim or Mormon serviceborg roleplay that allows for borgs to not contain alcohol.

## Why It's Good For The Game

All the same reasons as the previous. Service borgs should be able to make the same drinks as bartenders can. This PR does not make service borgs strictly better than bartenders, for a few reasons. For one, service borgs cannot serve harmful drinks in many cases even if someone asks for them. A bartender can, and many people like drinking deadly drinks. For two, bartenders can carry a shitload of containers that have reagants not in the booze or soda dispenser. A service cyborg only gets one beaker slot. Plus, the booze and soda dispenser can make far more drinks in terms of quantity at once.

## Changelog
:cl:
add: Nanotrasen has updated the standard-model service cyborg to include far more drinks! 
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagent_containers/borghydro.dm [hunk 1]
```diff
@@ -264,6 +261,37 @@ Borg Shaker
...
+/obj/item/reagent_containers/borghypo/borgshaker/alcoholic
+		name = "cyborg booze shaker"
+		desc = "An advanced alcoholic drink synthesizer and mixer."
+		icon = 'icons/obj/drinks.dmi'
+		icon_state = "shaker"
+		possible_transfer_amounts = list(5,10,20)
+		charge_cost = 20 //Lots of reagents all regenerating at once, so the charge cost is lower. They also regenerate faster.
+		recharge_time = 3
+		volume = 50
+		accepts_reagent_upgrades = FALSE
```
Comment (Mat05usz) [inline code/modules/reagents/reagent_containers/borghydro.dm]: Redundant code, as it's already defined on parent.

**Comment by Mat05usz** [comment]
Maybe try to find another icon for that (if possible), having two same looking shakers in the UI is kinda eeh

File: code/modules/reagents/reagent_containers/borghydro.dm [hunk 2]
```diff
@@ -174,8 +174,8 @@ Borg Hypospray
 Borg Shaker
 */
 /obj/item/reagent_containers/borghypo/borgshaker
-	name = "cyborg shaker"
-	desc = "An advanced drink synthesizer and mixer."
+	name = "cyborg soda shaker"
```
Comment (Mat05usz) [inline code/modules/reagents/reagent_containers/borghydro.dm:177]: At this point, maybe just create separate soda and alcoholic shaker, and leave borgshaker as a parent class for them. Keep the old name, move name/desc and reagents to the soda shaker.

File: code/modules/reagents/reagent_containers/borghydro.dm (hunk 1 above)
Comment (PowerfulBacon) [inline code/modules/reagents/reagent_containers/borghydro.dm]:
```suggestion
```

**Pull request closed without merging**